src/server/server_family.cc

```diff
@@ -1191,7 +1191,10 @@ void ServerFamily::Client(CmdArgList args, ConnectionContext* cntx) {
     return (*cntx)->SendBulkString(result);
   }
 
-  LOG_FIRST_N(ERROR, 10) << "Subcommand " << sub_cmd << " not supported";
+  if (sub_cmd != "SETINFO") {
```

Comment:
Can you please just add SETINFO command that replies OK ?
seems like equivalent effort and not special cases.

Reply:
I wrote about this in the comments, the reason I did not do this, is that I found it misleading, to send an `ok` reply for a subcommand we do not support. Since you think it's ok, I will make the change.